### PR TITLE
Follow one sentence per line in list items

### PR DESCRIPTION
**Pull request by roj1512**

### CODE REVIEW

File: site/docs/es/plugins/ratelimiter.md [hunk 1]
```diff
@@ -20,9 +20,11 @@ Este plugin expone 5 opciones personalizables:
 
 - `timeFrame`: El intervalo de tiempo durante el cual se monitorizarán las peticiones (por defecto es `1000` ms).
 - `limit`: El número de peticiones permitidas dentro de cada `timeFrame` (por defecto es `1`).
-- `storageClient`: El tipo de almacenamiento que se utilizará para mantener un registro de los usuarios y sus peticiones. El valor por defecto es `MEMORY_STORE` que utiliza un [Mapa] en memoria (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map), pero también se puede pasar un cliente Redis (más información en [About storageClient](#about-storageclient)).
+- `storageClient`: El tipo de almacenamiento que se utilizará para mantener un registro de los usuarios y sus peticiones.
+  El valor por defecto es `MEMORY_STORE` que utiliza un [Mapa] en memoria (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map), pero también se puede pasar un cliente Redis (más información en [About storageClient](#about-storageclient)).
```
Comment (KnorpelSenf) [inline site/docs/es/plugins/ratelimiter.md:24]:
```suggestion
  El valor por defecto es `MEMORY_STORE` que utiliza un [Mapa](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map) en memoria, pero también se puede pasar un cliente Redis (más información en [About storageClient](#about-storageclient)).
```

**Review by KnorpelSenf — CHANGES_REQUESTED** [review]

File: site/docs/es/plugins/ratelimiter.md (hunk 1 above)
Comment (roj1512) [reply site/docs/es/plugins/ratelimiter.md:24]: Awesome!

File: site/docs/es/plugins/ratelimiter.md [hunk 2]
```diff
@@ -20,9 +20,11 @@ Este plugin expone 5 opciones personalizables:
 
 - `timeFrame`: El intervalo de tiempo durante el cual se monitorizarán las peticiones (por defecto es `1000` ms).
 - `limit`: El número de peticiones permitidas dentro de cada `timeFrame` (por defecto es `1`).
-- `storageClient`: El tipo de almacenamiento que se utilizará para mantener un registro de los usuarios y sus peticiones. El valor por defecto es `MEMORY_STORE` que utiliza un [Mapa] en memoria (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map), pero también se puede pasar un cliente Redis (más información en [About storageClient](#about-storageclient)).
+- `storageClient`: El tipo de almacenamiento que se utilizará para mantener un registro de los usuarios y sus peticiones.
+  El valor por defecto es `MEMORY_STORE` que utiliza un [Mapa](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map) en memoria, pero también se puede pasar un cliente Redis (más información en [About storageClient](#about-storageclient)).
```
Comment (KnorpelSenf) [inline site/docs/es/plugins/ratelimiter.md:24]:
```suggestion
  El valor por defecto es `MEMORY_STORE` que utiliza un [Mapa](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map) en memoria, pero también se puede pasar un cliente Redis (más información en [About storageClient](#sobre-storageclient)).
```

**Review by KnorpelSenf — CHANGES_REQUESTED** [review]
Found one more.

**Review by KnorpelSenf — APPROVED** [review]
Thanks!

**Pull request merged**